python/tests/simplify.py

```diff
@@ -1,12 +1,12 @@
 # MIT License
 #
-# Copyright (c) 2018-2020 Tskit Developers
-# Copyright (c) 2015-2018 University of Oxford
+# Copyright(c) 2018 - 2020 Tskit Developers
```

Comment:
Lots of unwanted formatting changes here - your editor has gone power mad!

Reply:
I don't know what's happened here, eek 🤦

Reply:
I think I might have accidentally used clang-format on the Python code without realising I was in the wrong directory. Anyway, will fix